src/main/java/org/opensearch/ad/ml/ModelManager.java

```diff
@@ -707,51 +558,16 @@ public void maintenance(ActionListener<Void> listener) {
             .compact(true)
             .precision(Precision.FLOAT_32)
             .boundingBoxCacheFraction(AnomalyDetectorSettings.BATCH_BOUNDING_BOX_CACHE_RATIO)
-            // same with dimension for opportunistic memory saving
-            .shingleSize(shingleSize)
+            // for external shingling, rcf does not recognize shingle
+            .shingleSize(1)
```

Comment:
does this affect singles and multis?

Reply:
This affects single-stream real time detectors.  HCAD models' configuration is in EntityColdStarter.

Reply:
is this getPreviewResults called by multis?

Reply:
Sorry, I was looking at the wrong places.  Yes, this affects both single stream/HCAD preview.

Reply:
OK. So this shingleSize = 1 for the tRCF is a signal that shingling is external. So we are using external shingling in the HCAD preview as well? We are using internal shingling in HCAD right?

Reply:
(information question so that we are all on same page)

Reply:
yes, we are using external shingling in HCAD preview, but internal shingling in HCAD real time detectors.

Reply:
Wait this shingleSize will be passed to the forest, so we would have shingleSize as 1 again.